app/config/config_dev.yml

```diff
@@ -18,6 +18,15 @@ services:
         arguments: ["@rest_client"]
         tags:
             - { name: data_collector, template: "AppBundle:Manage:api-collector", id: "api-collector" }
+    mail_sender:
+        class: AppBundle\Service\Mailer\MailSenderMock
+        arguments: [ "@validator", "@snc_redis.default" ]
+        calls:
+            - [ addSwiftMailer, [ "default", "@mailer.swift_mailers.default"] ]
+    mailer.transport.smtp.default:
+        class: AppBundle\Service\Mailer\Transport\TransportMock
+    mailer.transport.smtp.secure:
```

Comment:
I dont' think this is needed now, all email is secure (inc default) there should be no reference to it in the code

Reply:
Good catch, done